src/main/java/org/llorllale/cactoos/matchers/Mismatches.java

```diff
@@ -53,9 +53,6 @@
  * @param <X> Type of item.
  * @param <M> Type of tested matcher.
  * @since 1.0.0
- * @todo #106:30min Convert all the Matcher tests to use Mismatches
```

Comment:
@andreoss have you properly validated that there are now zero test which could use `Mismatches`?

Reply:
@victornoel I've left those where `Throws` makes more sense, imo. Like `AssetionTest`, `MatcherEnvelopeTest`, and `ThrowsTest` itself.
There is also `new IsNot<>(new Matches<>(..))` idiom, but they are not equivalent because only boolean output is checked, not the messages.

Reply:
@andreoss the one with `new IsNot<>(new Matches<>(..))` were the primary target of this todo :) so yes trey should be replaced. Please simply add a todo to continue the work. Let's avoid extra big PR again please :)

Reply:
@andreoss sorry, I meant there were the primary target of `Mismatches`, not the todo itself :)